discord/message.go

```diff
@@ -213,6 +221,16 @@ func (s Sticker) TagList() []string {
 	return strings.Split(s.Tags, ",")
 }
 
+type StickerType int
+
+// https://discord.com/developers/docs/resources/sticker#sticker-object-sticker-types
+const (
+	// An official sticker in a pack, part of Nitro or in a removed purchasable pack
+	StickerTypeStandard StickerType = iota
+	// A sticker uploaded to a Boosted guild for the guild's members
+	StickerTypeGuild
```

Comment:
```suggestion
	// StandardSticker is an official sticker in a pack, part of Nitro or in a removed purchasable pack.
	StandardSticker StickerType = iota + 1
	// GuildSticker is a sticker uploaded to a boosted guild for the guild's members.
	GuildSticker
```
Go comments always start with the name of the entity.
Also, arikawa uses the convention of putting the name of the const's type (a shortened version, if appropriate) last, not first.
Lastly, the `StickerType`'s value starts at `1` not `0`, so one would need to use `iota + 1`, since `iota` starts at `0`.